projects/observability/src/shared/components/cartesian/cartesian-chart.component.scss

```diff
@@ -6,6 +6,25 @@
 }
 
 :host ::ng-deep {
+  .extra-legend-entry {
+    position: absolute;
+    display: flex;
+    gap: 2px;
+    margin-top: 6px;
+    align-items: center;
+    & > .extra-legend-box {
+      width: 16px;
+      height: 16px;
+      border: 2px dashed red;
+      border-radius: 50%;
+      background: repeating-linear-gradient(90deg, pink, white 6px, pink, white 6px);
```

Comment:
Instead of html pink, can we use one of the hues from the hypertrace color palette? `$purple-4` or `$red-5` maybe?